### PR TITLE
Add another kerberos realm

### PR DESCRIPTION
**Pull request by arhimondr**

### CODE REVIEW

File: prestodb/hdp2.6-hive-kerberized/files/etc/supervisord.d/kdc.conf [hunk 1]
```diff
@@ -1,13 +1,13 @@
 [program:krb5kdc]
-command=/bin/bash -c "exec /usr/sbin/krb5kdc -r LABS.TERADATA.COM -P /var/run/krb5kdc.pid -n"
+command=/bin/bash -c "exec /usr/sbin/krb5kdc -P /var/run/krb5kdc.pid -n -r LABS.TERADATA.COM -n -d /var/kerberos/krb5kdc/principal-other -r OTHERLABS.TERADATA.COM"
```
Comment (findepi) [inline prestodb/hdp2.6-hive-kerberized/files/etc/supervisord.d/kdc.conf]: nit: you can remove `/bin/bash -c "` and final `"`

as i understand this runs single KDC process for the two realms. Should we have two processes to better simulate multi-realm world?

File: prestodb/hdp2.6-hive-kerberized/files/etc/supervisord.d/kdc.conf [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 autostart=true
 autorestart=true
 redirect_stderr=true
 stdout_logfile=/dev/stdout
 stdout_logfile_maxbytes=0
 
 [program:kadmind]
-command=/bin/bash -c "exec /usr/sbin/kadmind -r LABS.TERADATA.COM -P /var/run/kadmind.pid -nofork"
+command=/bin/bash -c "exec /usr/sbin/kadmind -P /var/run/kadmind.pid -nofork -r LABS.TERADATA.COM"
```
Comment (findepi) [inline prestodb/hdp2.6-hive-kerberized/files/etc/supervisord.d/kdc.conf]: nit: you can remove `/bin/bash -c "` and final `"`

File: prestodb/hdp2.6-hive-kerberized/files/var/kerberos/krb5kdc/kdc.conf [hunk 3]
```diff
@@ -10,3 +10,16 @@
   admin_keytab = /var/kerberos/krb5kdc/kadm5.keytab
   supported_enctypes = aes128-cts:normal des3-hmac-sha1:normal arcfour-hmac:normal des-hmac-sha1:normal des-cbc-md5:normal des-cbc-crc:normal
  }
+
+ OTHERLABS.TERADATA.COM = {
+  #master_key_type = aes256-cts
```
Comment (findepi) [inline prestodb/hdp2.6-hive-kerberized/files/var/kerberos/krb5kdc/kdc.conf]: remove; also remove from `LABS.TERADATA.COM` realm

File: prestodb/hdp2.6-hive-kerberized/files/etc/krb5.conf [hunk 4]
```diff
@@ -13,6 +13,10 @@
 
 [realms]
  LABS.TERADATA.COM = {
-  kdc = hadoop-master
+  kdc = hadoop-master:88
+  admin_server = hadoop-master
+ }
+ OTHERLABS.TERADATA.COM = {
```
Comment (findepi) [inline prestodb/hdp2.6-hive-kerberized/files/etc/krb5.conf]: does the OTHERLABS's admin server allow to admin the OTHERLABS realm (like passwd)? `kadmind` configuration in supervisord doesn't say "OTHER"

File: prestodb/hdp2.6-hive-kerberized/Dockerfile [hunk 5]
```diff
@@ -23,6 +23,9 @@ COPY ./files /
 # CREATE KERBEROS DATABASE
 RUN /usr/sbin/kdb5_util create -s -P password
 
+# CREATE ANOTHER KERBEROS DATABASE
+RUN /usr/sbin/kdb5_util create -d /var/kerberos/krb5kdc/principal-other -r OTHERLABS.TERADATA.COM -s -P password
```
Comment (findepi) [inline prestodb/hdp2.6-hive-kerberized/Dockerfile]: nit: `123456` is a better password than `password` (at usage site, the meaning is easier to guess)

**Comment by arhimondr** [comment]
Merged. @phd1994 thanks for the contribution!

**Pull request merged**